src/console/widget.ts

```diff
@@ -114,6 +114,15 @@ class ConsoleWidget extends Widget {
     return widget;
   }
 
+  /*
+   * The last cell in a console is always a `CodeCellWidget` prompt.
```

Comment:
Not that I can see. I think that it makes sense to define the `prompt` as:

1. the final cell
2. a `CodeCellWidget`